spec/controllers/requests_controller_spec.rb

```diff
@@ -182,6 +182,10 @@
         expect(assigns(:request)).to be_persisted
       end
 
+      it "sends an email" do
+        expect{ post :create, request: valid_attributes, session: valid_session }.to change{ ActionMailer::Base.deliveries.count }.by(1)
+      end
+
```

Comment:
I would suggest to add a similar test, that no email is sent, when using invalid params